pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,511 @@
+    echo "${OPTION}=mysql_native_password" >> ${OUTPUT_FILE}
+    echo "${OPTION}=sha256_password" >> ${OUTPUT_FILE}
+    echo "${OPTION}=caching_sha2_password" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-change-buffering" ]; then
+    echoit "  > Adding possible values all, none, inserts, deletes, changes, purges for option '${OPTION}' to the final list..."
+    echo "${OPTION}=all" >> ${OUTPUT_FILE}
+    echo "${OPTION}=none" >> ${OUTPUT_FILE}
+    echo "${OPTION}=inserts" >> ${OUTPUT_FILE}
+    echo "${OPTION}=deletes" >> ${OUTPUT_FILE}
+    echo "${OPTION}=changes" >> ${OUTPUT_FILE}
+    echo "${OPTION}=purges" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-default-row-format" ]; then
+    echoit "  > Adding possible values dynamic, compact, redundant for option '${OPTION}' to the final list..."
+    echo "${OPTION}=dynamic" >> ${OUTPUT_FILE}
+    echo "${OPTION}=compact" >> ${OUTPUT_FILE}
+    echo "${OPTION}=redundant" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--log-output" ]; then
+    echoit "  > Adding possible values FILE, TABLE, NONE for option '${OPTION}' to the final list..."
+    echo "${OPTION}=FILE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=TABLE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=NONE" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--master-info-repository" ]; then
+    echoit "  > Adding possible values FILE, TABLE for option '${OPTION}' to the final list..."
+    echo "${OPTION}=FILE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=TABLE" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--relay-log-info-repository" ]; then
+    echoit "  > Adding possible values FILE, TABLE for option '${OPTION}' to the final list..."
+    echo "${OPTION}=FILE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=TABLE" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--windowing-use-high-precision" ]; then
+    echoit "  > Adding possible values 0, 1 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=0" >> ${OUTPUT_FILE}
+    echo "${OPTION}=1" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--default-storage-engine" ]; then
+    echoit "  > Adding possible value InnoDB for option '${OPTION}' to the final list..."
+    echo "${OPTION}=InnoDB" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--binlog-row-metadata" ]; then
+    echoit "  > Adding possible value full and minimal for option '${OPTION}' to the final list..."
+    echo "${OPTION}=full" >> ${OUTPUT_FILE}
+    echo "${OPTION}=minimal" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--binlog-transaction-dependency-tracking" ]; then
+    echoit "  > Adding possible value COMMIT_ORDER , WRITESET and WRITESET_SESSION for option '${OPTION}' to the final list..."
+    echo "${OPTION}=COMMIT_ORDER" >> ${OUTPUT_FILE}
+    echo "${OPTION}=WRITESET" >> ${OUTPUT_FILE}
+    echo "${OPTION}=WRITESET_SESSION" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--caching-sha2-password-public-key-path" ]; then
```

Comment:
Why this is not removed, as discussed yesterday ?

Reply:
fixed